src/components/Coaching/CoachingDetail/coachingMocks.ts

```diff
@@ -0,0 +1,100 @@
+export const activitySummaryMocks = {
```

Comment:
If it were me, I would wrap this in a `{ ActivitySummary: ... }` object and `appointmentResultsMocks` in a `{ AppointmentResults: ... }` object (and so on for other mocks) so that tests can use it directly like `mocks={activitySummaryMocks}` or `mocks={ ...activitySummaryMocks, ...appointmentResultsMocks }`. It's really easy to use the wrong operation name, so centralizing them means it's a little harder to mess up.